src/shared/components/oncoprint/DeltaUtils.ts

```diff
@@ -1113,11 +1126,7 @@ function transitionGeneticTrack(
             description: nextSpec.oql,
             data_id_key: 'uid',
             data: nextSpec.data,
-            tooltipFn: makeGeneticTrackTooltip(
```

Comment:
tooltipFn already returns a string?  if so, that's convenient!

Reply:
yes, a string or jquery